extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/linkHandler/MediaCCCConferenceLinkHandlerFactory.java

```diff
@@ -14,19 +18,43 @@ public String getUrl(String id, List<String> contentFilter, String sortFilter) t
     }
 
     @Override
-    public String getId(String url) throws ParsingException {
-        if(url.startsWith("https://api.media.ccc.de/public/conferences/")) {
-            return url.replace("https://api.media.ccc.de/public/conferences/", "");
-        } else if(url.startsWith("https://media.ccc.de/c/")) {
-            return Parser.matchGroup1("https://media.ccc.de/c/([^?#]*)", url);
-        } else {
-            throw new ParsingException("Could not get id from url: " + url);
+    public String getId(String urlString) throws ParsingException {
+        if (urlString.startsWith("https://api.media.ccc.de/public/conferences/")) {
+            return urlString.replace("https://api.media.ccc.de/public/conferences/", "");
+        } else if (urlString.startsWith("https://media.ccc.de/c/")) {
+            return Parser.matchGroup1("https://media.ccc.de/c/([^?#]*)", urlString);
         }
+
+        URL url;
+        try {
+            url = Utils.stringToURL(urlString);
+        } catch (MalformedURLException e) {
+            throw new IllegalArgumentException("The given URL is not valid");
+        }
+
+        String path = url.getPath();
+        // remove leading "/" of URL-path if URL-path is given
+        if (!path.isEmpty()) {
+            path = path.substring(1);
+        }
+
+        if (path.contains("b/")) {
```

Comment:
It looks like you want to check if the path starts with `b/`. if this is the case use
```suggestion
        if (path.startsWith("b/")) {
```
for better performance. In which case can an url path start with `b/`? Do we need to check the host here or do we have only urls like `/b/myConferenceId`?

Reply:
Instead of this, just do `urlString.startsWith("https://media.ccc.de/b/")` (and preferably change the name of the `urlString` variable back to just `url`)

Reply:
Yes it's conferences @TobiGr 
I changed back to how it was and added ``|| url.startsWith("https://media.ccc.de/b/")`` in the if condition